packages/trader/src/AppV2/Components/AccumulatorStats/accumulator-stats.scss

```diff
@@ -0,0 +1,191 @@
+            display: flex;
+            width: calc(100% - 48px - 9px - 40px);
+            justify-content: space-between;
+            gap: var(--core-spacing-400);
+            position: relative;
+            overflow-x: auto;
+            &::-webkit-scrollbar {
+                display: none;
+            }
+            &__moving {
+                display: flex;
+                align-items: center;
+                gap: var(--core-spacing-400);
+            }
+
+            &__stat {
+                position: relative;
+                padding: var(--core-spacing-50) var(--core-spacing-400);
+                margin-top: var(--core-spacing-200);
+                p {
+                    position: relative;
+                    display: inline-block;
+                    overflow: hidden;
+                    margin: 0;
+                    &:after {
+                        content: '';
+                        position: absolute;
+                        bottom: 0;
+                        left: 0;
+                        width: 100%;
+                        height: var(--core-borderRadius-100);
+                        transform: scaleX(0);
+                        transform-origin: left;
+                        background-color: var(--core-color-solid-green-600);
+                        opacity: 0;
+                    }
+                }
+                .animate-error:after {
+                    @include accumulatorUnderlineAnimation(var(--core-color-solid-red-600));
+                }
+                .animate-success:after {
+                    @include accumulatorUnderlineAnimation;
+                }
+            }
+        }
+        &__expand {
+            background: transparent;
+            border: none;
+            padding-top: var(--core-spacing-200);
+        }
+    }
+}
+
+.accumulator-stats-sheet-wrapper {
+    .stats-sheet {
+        padding: var(--core-spacing-800);
+        text-align: center;
+        .slide-right {
+            @include accumulatorSlideToNext;
+        }
+
+        &__title {
+            margin-bottom: var(--core-spacing-1200);
+        }
+        &__caption {
+            margin-bottom: var(--core-spacing-400);
+        }
+        &__stats {
+            display: flex;
+            padding: var(--core-spacing-600) var(--core-spacing-75);
+            justify-content: space-between;
+            border-bottom: var(--core-spacing-75) solid var(--core-color-opacity-black-100);
+            &__moving {
+                display: flex;
+                align-items: center;
+            }
+            &__stat {
+                width: var(--core-spacing-3600);
+                text-align: center;
+                position: relative;
+                p {
+                    display: inline-block;
+                    width: auto;
+                    position: relative;
+                    &:after {
+                        content: '';
```

Comment:
Hm, looks like something from line 83)

Reply:
Actually one is on screen and the other one is on portal . So don't have any common ground for it to extract . Not sure if making it a mixin would be a better option